src/test/functional/features/opening-hours.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Opening Hours
+
+  Background:
+    Given I am on new browser
+    Given I am on FACT homepage
+    And I am on the admin portal sign in page
+    When I fill in the Username and Password fields with my authenticated credentials "hmcts.super.fact@gmail.com" "Pa55word11"
```

Comment:
I think Kian had a security vulnerability around exposing sensitive data. This would kind of fall under that, although it may not necessarily be picked up by the security team. 

Is it possible to use environment variables here to pick up these values, or something similar? Might be one we need to take offline and fix another time, since it's a known wider issue among the team that we all know about

Reply:
This exists in almost all of the other functional test files. It's also in the readme file and displays in github. I agree we should fix it but I think a separate ticket would be needed.

Reply:
Hi gents, just on this security issue - for the API we’ve deployed the fixes utilising the AZ vault with terraform. We can deploy a similar fix here, as @agraham01 says it is definitely for another ticket though. We will need to get this addressed before go live.